e2e/specs/ward-to another-ward.spec.ts

```diff
@@ -0,0 +1,81 @@
+// specs/ward-allocation.spec.ts
+
+import { test, expect } from '@playwright/test';
+import { WardAllocation } from '../pages/ward-allocation';
+import { type BedStatus } from '../types';
+
+test.beforeEach(async ({ page }) => {
+  const wardPage = new WardAllocation(page);
+  await wardPage.goto('bed-management');
+});
+
+test('create  beds and wards', async ({ page }) => {
+  const wardPage = new WardAllocation(page);
+
+  const ward1BedType = {
+    name: 'ward1-bed',
+    displayName: 'Ward 1 Bed',
+    description: 'Standard bed in Ward 1',
+  };
+
+  const ward1Status: BedStatus = 'AVAILABLE';
+  const ward1BedLayout = {
+    bedId: 301,
+    bedNumber: '301A',
+    bedUuid: 'uuid-ward1-bed',
+    rowNumber: 1,
+    columnNumber: 1,
+    location: 'ward1',
+    status: ward1Status,
+    bedType: ward1BedType,
+  };
+
+  await wardPage.allocateWard(ward1BedLayout);
```

Comment:
Should the flow be that we create the bed types first, then allocate beds, and then verify the results?